lesson16/main.js

```diff
@@ -125,6 +127,28 @@ function createNewIconContainer(article) {
   return newIconContainer;
 }
 
+function createCommentIconContainer(article) {
+  const commentIconContainer = document.createElement("div");
+  commentIconContainer.classList.add("comment-container");
+  if (article.comment) {
+    const commentIcon = document.createElement("img");
+    commentIcon.classList.add("comment-icon");
+    commentIcon.src = "./img/comment.png";
+    commentIcon.width = "14";
+    commentIcon.height = "14";
+    commentIconContainer.appendChild(commentIcon);
+
+    const numOfComments = document.createElement("div");
+    numOfComments.classList.add("comment-num");
+    const numOfCommentProps = Object.keys(article.comment).length;
+    numOfComments.textContent = numOfCommentProps;

```

Comment:
IMO
I didn't see much need to make it variable.
```suggestion
    numOfComments.textContent = article.comment.length;
```

Reply:
thank you  sae-san :)
I fixed that.
2121a30595dfb8140a2fa889fdbbaeff2444f923